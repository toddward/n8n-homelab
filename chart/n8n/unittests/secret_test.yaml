suite: test secret

templates:
  - secret.yaml

tests:
  - it: should match snapshot of default values
    set:
      encryptionKey: my-secret-key
    release:
      name: n8n
      namespace: n8n
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    asserts:
      - matchSnapshot: { }
