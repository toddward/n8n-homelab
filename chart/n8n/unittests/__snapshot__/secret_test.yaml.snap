should match snapshot of default values:
  1: |
    apiVersion: v1
    data:
      N8N_ENCRYPTION_KEY: bXktc2VjcmV0LWtleQ==
    kind: Secret
    metadata:
      annotations:
        helm.sh/hook: pre-install
        helm.sh/hook-delete-policy: before-hook-creation
      labels:
        app: n8n
        chart: n8n-1.0.0
        heritage: Helm
        release: n8n
      name: n8n-encryption-key-secret
      namespace: n8n
    type: Opaque
